tests/unit/test_compat.py

```diff
@@ -125,8 +125,7 @@ def test_console_to_str_warning(monkeypatch):
     some_bytes = b"a\xE9b"
 
     def check_warning(msg, *args, **kwargs):
-        assert msg.startswith(
-            "Subprocess output does not appear to be encoded as")
+        assert args[0] == 'Subprocess output'
```

Comment:
Would it be possible to check for “encoded as” here? “Subprocess output” is too vague here and may produce false positive.

Reply:
TBH I don't even know where the `Subprocess output` thing come from :smile:  I'll check the message as well just in case.